eng/scripts/Language-Settings.ps1

```diff
@@ -17,25 +16,30 @@ function Get-java-PackageInfoFromRepo ($pkgPath, $serviceDirectory, $pkgName)
     $pkgVersion = $projectData.project.version
     $pkgGroup = $projectData.project.groupId
 
-    if ($projectPkgName -eq $pkgName)
+    if ($pkgName -and ($projectPkgName -ne $pkgName))
     {
-        $pkgProp = [PackageProps]::new($pkgName, $pkgVersion.ToString(), $pkgPath, $serviceDirectory, $pkgGroup)
-        if ($projectPkgName -match "mgmt" -or $projectPkgName -match "resourcemanager")
-        {
-          $pkgProp.SdkType = "mgmt"
-        }
-        elseif ($projectPkgName -match "spring")
-        {
-          $pkgProp.SdkType = "spring"
-        }
-        else
-        {
-          $pkgProp.SdkType = "client"
-        }
-        $pkgProp.IsNewSdk = $pkgGroup.StartsWith("com.azure")
-        $pkgProp.ArtifactName = $pkgName
-        return $pkgProp
+      return $null
+    }
+
+    $pkgProp = [PackageProps]::new($projectPkgName, $pkgVersion.ToString(), $pkgPath, $serviceDirectory, $pkgGroup)
+    if ($projectPkgName -match "mgmt" -or $projectPkgName -match "resourcemanager")
+    {
+      $pkgProp.SdkType = "mgmt"
+    }
+    elseif ($projectPkgName -match "spring")
+    {
+      $pkgProp.SdkType = "spring"
+    }
+    else
+    {
+      $pkgProp.SdkType = "client"
+    }
+    $pkgProp.IsNewSdk = $False
+    if ($pkgGroup) {
```

Comment:
There shouldn't be any cases where groupid isn't set. Did you hit a case?

Reply:
Yes i did.

Reply:
For my information do you remember what that case was?